src/OrchardCore/OrchardCore.Abstractions/OrchardCoreConstants.cs

```diff
@@ -0,0 +1,8 @@
+namespace OrchardCore;
+
+public class OrchardCoreConstants
```

Comment:
@hishamco I like what you did here but I have a suggestion.

If you want to call this file `OrchardCoreConstats` I would suggest that you sub section it to accommodate other contacts for example 

```csharp
public class OrchardCoreConstants
{
    public class Shell
    {
        public const string TenantFileName = "tenant.json";
    }


    public class Role
    {
        public const string Administrator = "Administrator";

    }
}
```


If you don't like that idea, then you should rename this file to `ShellConstants` instead.

Reply:
There's no role here, you might mean application settings

Reply:
I meant role when we add role constants.

The name OrchardCoreConstants is very vague to use here unless you want to put all the commons constants here like suggested above

Reply:
> The name OrchardCoreConstants is very vague

Right, but we usually name the constant class based on the project, your suggestion then we can finalize this PR

Reply:
My suggestion is here https://github.com/OrchardCMS/OrchardCore/pull/13438#discussion_r1477125957

Reply:
While I don't have a role in my PR, I used the `Shell` class also, added a `Configuration` one. Did I miss anything?